README_GITFLEXIMOD.rst

```diff
@@ -122,6 +110,6 @@ locally unless you explicitly push them up to GitHub. Feel free to
 create whatever local branches you'd like.) For example::
 
   git checkout -b my_ctsm_branch
-  git add Externals.cfg
+  git add .gitmodules
```

Comment:
.gitmodules should already be in the repository, you don't need to add it.

Reply:
Correct me if I'm wrong: I think this describes the workflow for changing .gitmodules and then committing it.

Reply:
My mistake - I just use the -a option to commit, i forgot that you can do this in two steps.  It's fine as is.